hpat/hiframes/api.py

```diff
@@ -1763,3 +1763,68 @@ def _analyze_op_pair_first(self, scope, equiv_set, expr):
 
 
 numba.array_analysis.ArrayAnalysis._analyze_op_pair_first = _analyze_op_pair_first
+
+
+def _append(A, B):
+    return None
+
+
+@overload(_append)
+def _append_overload(A, B):
```

Comment:
I don't think this is a good place for hidden function used nowhere in `hiframes`.
Why do you need it overloaded?